packages/cannoli-core/src/nl-generator/compiler.ts

```
@@ -0,0 +1,531 @@
+					label: choiceLabel,
+				};
+
+			case "list":
+				return {
+					...baseEdge,
+					color: "5", // Cyan
+					label: edge.label,
+				};
+
+			case "chat":
+				let chatLabel = edge.label || "";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)